frontend/src/pages/AccountSend.vue

```diff
@@ -866,6 +866,22 @@ function useSendForm() {
     return true;
   }
 
+  function isValidBatchSendAmount(val: string | undefined, tokenInput?: TokenInfoExtended | null | undefined) {
+    const tokenToUse = tokenInput || token.value;
+    if (!tokenToUse) return tc('Send.select-a-token');
+
+    const { address: tokenAddress, decimals } = tokenToUse;
+
+    // Check total Batch Send amount
+    const totalBatchSendAmount = summaryAmount.value.get(tokenToUse);
+    if (totalBatchSendAmount) {
+      const totalAmount = parseUnits(totalBatchSendAmount, decimals);
+      if (totalAmount.gt(balances.value[tokenAddress])) return `${tc('Send.total-amount-exceeds-balance')}`;
+    }
+
+    return isValidTokenAmount(val, tokenInput);
```

Comment:
Can we add comments to document the input args and purpose of this method? At first I thought `val` was intended to be the total batch send amount and the method was supposed to validate the full batch send. But then I thought it's supposed to be an individual send amount for the sole purpose of being passed to `isValidTokenAmount` to validate an individual row? But actually, it looks like it does both? 

Maybe I'm missing some context, but there's no accompanying issue or PR description, and IMO there should always be at least one :)

Reply:
Absolutely. Linked an issue and updated PR description. Added comments to the code block in https://github.com/ScopeLift/umbra-protocol/pull/597/commits/218b5c6c5171bc9b8e044c8c049f2d57cddb495c